lib/ruby2d/text.rb

```diff
@@ -11,6 +11,10 @@ def initialize(x=0, y=0, text="Hello World!", size=20, font=nil, c='white', z=0)
       
       # if File.exists? font
         @font = font
+      end
+      
+      # if File.exists? font
+        # @font = font
       # else
```

Comment:
Aw man, I think I migrated old code incorrectly and it's not right now. Later in the evening I will work on this and reimplement the feature. Sorry to waste your time like that!